### ISSUE TITLE
Plot fails with zero columns and just an index in the dataframe

### ISSUE THREAD

**Issue by ElDeveloper** [biocore/emperor#700]
See this forum thread:
https://forum.qiime2.org/t/emperor-plot-stuck-with-550-samples/

**Comment by jwdebelius** [comment]
Hi,

I'm getting a similar issue. The short version is that I get an Emperor object without a drop down for metadata if I use the `Emperor` class, but no problem when I use the QIIME2 python API. 

<img width="572" alt="screen shot 2018-11-20 at 3 05 57 pm" src="https://user-images.githubusercontent.com/5534846/48779102-20f74580-ecd7-11e8-8081-2b499e9612b1.png">

I'm running version 1.0.0b17 in a QIIME 2-2018.8 installation.

I load the mapping file and pre-calculated distance matrices. The mapping file gets converted to a QIIME 2 object, and I calculate the PCoA in qiime2 (because then its easier to pass into the qiime2 viewer object). 

```python
q2_map = Metadata(s_map)
pc_art = \
    diversity.methods.pcoa(distance_matrix=dm_art).pcoa
```
If I run 
```python
bc_emp = emperor.visualizers.plot(pc_art, metadata=q2_map).visualization
bc_emp
```
I get a fully functional emperor visualisation (although possibly without some of the methods from the Emperor object to programatically set parameters)

If I run,

```python
e_ = Emperor(bc_pc_art.view(skbio.OrdinationResults), s_map, remote=True)
```
I get an object without the metadata categories (see attached). I've got a dataset with about 1000 samples.

Thanks,
Justine

**Comment by ElDeveloper** [comment]
As you note, this issue only happens when using the Emperor object directly in the Jupyter notebook. This bug was fixed in the 1.0.0b18 release (has to do with namespacing of the JavaScript dependecies and a new version of the Jupyter notebook).

You can upgrade emperor (pip install -U emperor --pre) or update your QIIME installation to the latest release. This was fixed in #694 :+1:

**Comment by jwdebelius** [comment]
@ElDeveloper: Thanks! I will try the emperor update.
